src/Tribe/Tracker.php

```diff
@@ -358,8 +358,19 @@ public function track_taxonomy_term_changes( $object_id, $terms, $tt_ids, $taxon
 
 		$tracked_post_types = $this->get_post_types();
 
-		$post = get_post( $object_id );
-		if ( empty( $post ) || ! in_array( $post->post_type, $tracked_post_types ) ) {
+		global $wpdb;
+		$post_id = $wpdb->get_var(
```

Comment:
I really would love to have this on a `tribe_post_exists()` template function. Would be super Handy.